README.md

```diff
@@ -102,6 +102,9 @@ Building has been tested with [Apache Maven 3.2](http://maven.apache.org/downloa
     [INFO] Final Memory: 22M/309M
     [INFO] ------------------------------------------------------------------------
 
+If you experience issues building on Mac OSX, please use the following command
```

Comment:
This isn't unique to Mac OSX, it happens for me on Linux, and I imagine it would happen on other platforms also.